test/test.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+'use strict';
```

Comment:
> Should the test check the API with the same semantics as users will use it in scripts?

The `use strict` in the beginning of the test ensures that the `test.js` itself conforms to the strict mode; it doesn't imply anything about the puppeteer code. We also have a pretty tight eslint rules that should enforce us into good code practices, so `use strict` in the `test.js` doesn't give us anything.